deploy/charts/library/templates/_cluster-role.tpl

```diff
@@ -147,4 +147,5 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumeclaims"]
     verbs: ["get", "update", "delete", "list"]
+---
```

Comment:
Let's revert this since I doubt it's needed.

Reply:
Missed it while cleaning up, removed it now.